### PR TITLE
Feature/hostname in stackdriver logName

### PR DESCRIPTION
**Pull request by brandoconnor**
## What is this change?

1. An enhancement to logging such that `logName` within StackDriver contains the hostname of the GCE instance.
2. minor formatting fixes around the edges.

## Why make this change?

This change helps engineers understand where logs are coming from.

### CODE REVIEW

**Comment by brandoconnor** [comment]
The implementation is opinionated but this gives some flexibility to the caller, allowing them to append the `logName` as they see fit. Resolves this issue: https://github.com/openx/ox-bqpipeline/issues/2

**Comment by brandoconnor** [comment]
Tests added.

Heh! I didn't see this PR until I submitted but looks like @shubhanan-ox is up to very similar here: https://github.com/openx/ox-bqpipeline/pull/4/files?file-filters%5B%5D=.py#diff-62db0910cd8a1b101462d1cb96406a21R56

File: tests/test_bqpipeline.py [hunk 1]
```diff
@@ -54,8 +63,24 @@ def test_create_job_config_destination(self):
...
+        bq = BQPipeline(
+            job_name='testjob', default_project='testproject', default_dataset='testdataset')
         cfg = bq.create_job_config(batch=False, create=False, overwrite=False)
         self.assertEqual(cfg.priority, bigquery.QueryPriority.INTERACTIVE)
-        self.assertEqual(cfg.create_disposition, bigquery.job.CreateDisposition.CREATE_NEVER)
-        self.assertEqual(cfg.write_disposition, bigquery.job.WriteDisposition.WRITE_EMPTY)
+        self.assertEqual(cfg.create_disposition,
+                         bigquery.job.CreateDisposition.CREATE_NEVER)
+        self.assertEqual(cfg.write_disposition,
+                         bigquery.job.WriteDisposition.WRITE_EMPTY)
```
Comment (brandoconnor) [inline tests/test_bqpipeline.py]: Bleh sorry about all the formatting above. Editor strikes again :shakes fist:

File: README.md [hunk 2]
```diff
@@ -74,10 +74,21 @@ source venv/bin/activate
...
 
+```bash
+pipenv install --python 3
 ```
-python3 -m pip install -r requirements.txt
+
+### Install with pipenv
+
+```bash
+pipenv run python -m unittest discover
```
Comment (jaketf) [inline README.md]: This seems like a duplicate of below.
@shubhanan-ox 
```suggestion
python3 -m pip install -r requirements.txt
```

**Comment by jaketf** [comment]
Closing this as it was looped into #4 by #6.

**Pull request closed without merging**